ui/src/components/list/group-view.js

```diff
@@ -1,15 +1,17 @@
 import { Baseline } from '../../shared/composers';
 import { colors } from '../../shared/constants';
 import { remcalc } from '../../shared/functions';
-import { raw as View } from './view';
+import View from './view';
 import styled from 'styled-components';
 
 const GroupView = styled(View)`
   display: block;
+
   padding-top: ${remcalc(62)};
```

Comment:
Could be written shorthand :
`padding: ${remcalc(62, 23, 5, 23)}`
Me just being picky